.github/CODEOWNERS

```
@@ -43,6 +43,7 @@
 /package.json                                               @keycloak/ui-maintainers
 /pnpm-lock.yaml                                             @keycloak/ui-maintainers
 /pnpm-workspace.yaml                                        @keycloak/ui-maintainers
+/themes/                                                    @keycloak/ui-maintainers
```

I agree that all files in here are UI team based. Still I wonder if want other maintainers to help with the translation files in `messages*.properties`. My suggestion below would allow either a UI maintainer or an overall maintainer to approve a change. 

```suggestion
/themes/                                                    @keycloak/ui-maintainers @keycloak/maintainers
```

---

Would that not require a review from both rather than either?

---

From what I understand, having two entries would notify both groups, but only one needs to approve. This is how I recall it from the cloud native entries.

---

Yes, for cloud-native I can confirm there's a logical OR. Either someone from `@keycloak/cloud-native-maintainers` approves, or a global maintainer. Which is the desired behaviour (for Cloud Native team).

---

Got it, made the change.